docs/content/en/install/local/_index.md

```diff
@@ -40,11 +40,10 @@ As K8ssandra deploys on a K8s cluster, one must be available to target for insta
 
 K8ssandra works with the following versions of Kubernetes either standalone or via a cloud provider:
 
-* 1.16
-* 1.17
+* 1.17 (minimum supported version)
 * 1.18
 * 1.19
-* 1.20
+* 1.20 and later Kubernetes releases
```

Comment:
If you would like to be more specific for the upper bound version, we are testing up to `v1.21.1` of k8s at current. 

Perhaps we remove the 'and later ...' and make the upper bound v1.21.1?

There are some newer versions (v.1.21.[2-4] and v1.22.1 that we haven't officially tested against.
As reference, k8s patch releases are [here](https://kubernetes.io/releases/patch-releases/#1-22).

Reply:
Inc'd that suggestion - thanks Jeff